src/site/content/en/react/how-to-use-amp-in-nextjs/index.md

```diff
@@ -0,0 +1,208 @@
+---
```

Comment:
staged doc: https://deploy-preview-1705--web-dev-staging.netlify.com/how-to-use-amp-in-nextjs